docs/management/advanced-options.asciidoc

```diff
@@ -268,7 +268,8 @@ the `xpack.banners.backgroundColor` configuration property.
 
 [horizontal]
 [[xpackdashboardmode-roles]]`xpackDashboardMode:roles`::
-**Deprecated. Use <<kibana-feature-privileges,feature privileges>> instead.**
+deprecated:[7.7.0]
+**Use <<kibana-feature-privileges,feature privileges>> instead.**
```

Comment:
This ends up looking a little funny because the link doesn't get bolded and the note doesn't seem connected to the deprecation info. Consider adding a line break and adding the original "Deprecated. use <<feature privileges>> instead>" language as the second sentence so the actual intent is clearer.

Feel free to ignore and ship as is if you disagree!

<img width="856" alt="Screenshot 2024-06-17 at 1 02 42 PM" src="https://github.com/elastic/kibana/assets/58563081/9dc5b293-2829-464b-8bd4-db5ec05852df">

Reply:
Ideally the "use xxx instead" would appear in the pop-up, but this table format seems to cut off that pop-up unfortunately. I'll add back the deprecated message like you suggest.

Reply:
Edited formatting in 945b1796ddd2cf5dcdd9c6988e9358efe9a6ffdd